tsl/src/continuous_aggs/invalidation_threshold.c

```diff
@@ -151,6 +181,48 @@ invalidation_threshold_scan_update(TupleInfo *ti, void *const data)
 	return SCAN_CONTINUE;
 }
 
+/*
+ * Get the invalidation threshold for the hypertable.
+ *
+ * This will also lock the row.
+ */
+int64
+invalidation_threshold_get(int32 hypertable_id, Oid type)
```

Comment:
I don't see a place where this threshold is ever used. Is the function only used to lock the tuple? Maybe we should just stick to a lock function for now then?

```suggestion
void
invalidation_threshold_lock(int32 hypertable_id, Oid type)
```

Reply:
It is currently used only for locking the tuple, but it seems prudent to have a get function since it is quite likely that you might want to print it out and use it in some way if you have locked it.